gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -396,7 +408,8 @@ private static Storage createStorage(
     HttpTransport httpTransport =
         HttpTransportFactory.createHttpTransport(
             options.getProxyAddress(), options.getProxyUsername(), options.getProxyPassword());
-    return new Storage.Builder(httpTransport, JSON_FACTORY, httpRequestInitializer)
+    return new Storage.Builder(
+            httpTransport, JSON_FACTORY, setGcsRequestTracker(options, httpRequestInitializer))
```

Comment:
I think we agreed to update the constructors instead of this method. That was the intent in adding the delegate to the RetryHTTPInitializer

Reply:
Updated